### PR TITLE
[ENH] The Guardian

### PR DESCRIPTION
**Pull request by nikicc**
New widget for fetching articles from The Guardian.

### CODE REVIEW

File: orangecontrib/text/widgets/owguardian.py [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+class OWGuardian(OWWidget):
+    class CredentialsDialog(OWWidget):
+        name = 'The Guardian Credentials'
+        want_main_area = False
+        resizing_enabled = False
+        cm_key = CredentialManager('The Guardian API Key')
+        key_input = ''
+
+        class Error(OWWidget.Error):
+            invalid_credentials = Msg('This credentials are invalid.')
```
Comment (ajdapretnar) [inline orangecontrib/text/widgets/owguardian.py]: *These

Comment (nikicc) [inline orangecontrib/text/widgets/owguardian.py]: Tnx, fixed :)

File: orangecontrib/text/widgets/owguardian.py [hunk 2]
```diff
@@ -0,0 +1,203 @@
...
+            invalid_credentials = Msg('These credentials are invalid.')
+
+        def __init__(self, parent):
+            super().__init__()
+            self.parent = parent
+            self.api = None
+
+            form = QFormLayout()
+            form.setContentsMargins(5, 5, 5, 5)
+            self.key_edit = gui.lineEdit(self, self, 'key_input', controlWidth=350)
```
Comment (kernc) [inline orangecontrib/text/widgets/owguardian.py]: Could we have `self.key_edit.setText('test')`. This key [seems to work](https://content.guardianapis.com/search?q=debates&api-key=test) for testing purposes ...

Comment (nikicc) [inline orangecontrib/text/widgets/owguardian.py]: Cool suff! I set it as the default value.

**Comment by nikicc** [comment]
@ajdapretnar I think this is ready for some testing.

**Comment by ajdapretnar** [comment]
So far so good, but I'm wondering whether we could not extract images from html as we did for Twitter? But we _could_ make this a part of another PR. :)

**Comment by nikicc** [comment]
I was thinking about this and we probably could. But let's put this in another PR.

**Comment by ajdapretnar** [comment]
Then, judging from my spastic clicking, this widget works. 😄

**Pull request merged**